dpc/src/account/address.rs

```diff
@@ -82,8 +82,8 @@ impl<C: Parameters> Address<C> {
     }
 
     /// Returns the address as an encryption public key.
-    pub fn to_encryption_key(&self) -> &<C::AccountEncryptionScheme as EncryptionScheme>::PublicKey {
-        &self.encryption_key
+    pub fn to_encryption_key(self) -> <C::AccountEncryptionScheme as EncryptionScheme>::PublicKey {
```

Comment:
Is the `PublicKey` `Copy`? If not, we should rename this to `into`, as per the API guidelines

Reply:
Good catch